### PR TITLE
Adding markers to my map

### PR DESCRIPTION
**Pull request by Yowzel**
just added markers to my map, best coffee shop in my town :)

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -55,8 +55,147 @@ function deleteTask(task) {
...
+//       {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+// }
+
 function createMap() {
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
-}
+
+        // Create a new StyledMapType object, passing it an array of styles,
```
Comment (dcollier) [inline portfolio/src/main/webapp/script.js:60]: It looks like the formatting in this function is off. I think if you bring everything in this function to the left by 4 spaces, you should be fine.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -55,8 +55,147 @@ function deleteTask(task) {
 }
 
 /** Creates a map and adds it to the page. */
+// function createMap() {
```
Comment (dcollier) [inline portfolio/src/main/webapp/script.js]: you can delete these commented out lines :)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -29,9 +29,11 @@ <h1>Gia-Huy's Portfolio</h1>
     </form>
     </hr>
     <ul id="quote-list"></ul>
-
+
```
Comment (dcollier) [inline portfolio/src/main/webapp/index.html]: can you remove the extra spaces in this file as well? (line 32 and 36)

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (Yowzel) [reply portfolio/src/main/webapp/script.js:60]: Hi! I fixed this in my new pull request info window :) . I was trying to fix this and push it to this branch but it was not letting me. Idk why

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (Yowzel) [reply portfolio/src/main/webapp/script.js]: I removed them in info window pr :)

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Yowzel) [reply portfolio/src/main/webapp/index.html]: I fixed this in info window pr :)

**Pull request merged**